### PR TITLE
crashes: increment the number of tracked ones

### PR DESCRIPTION
**Pull request by matthiaskrgr**
r? @jieyouxu

### CODE REVIEW

**Comment by jieyouxu** [comment]
Thanks! @bors r+ rollup

**Comment by workingjubilee** [comment]
@bors r-

Conflicts with https://github.com/rust-lang/rust/pull/125466

File: tests/crashes/125432.rs
Comment (workingjubilee) [inline tests/crashes/125432.rs:1]: Please delete this.

File: tests/crashes/125520.rs
Comment (workingjubilee) [inline tests/crashes/125520.rs:1]: This one too, please.

File: tests/crashes/125370.rs
Comment (workingjubilee) [inline tests/crashes/125370.rs:1]: If you would.

**Review by workingjubilee — COMMENTED** [review]
This has even more conflicts, actually, a bunch of these crashes need to go away:

https://github.com/rust-lang/rust/pull/125592#issuecomment-2132417811

**Comment by jieyouxu** [comment]
> lol @jieyouxu fwiw if I set something to rollup=iffy there is usually a good reason for it:D

Oh sorry, I missed the rollup=iffy comment...

**Comment by matthiaskrgr** [comment]
I've added the last couple of new ices, removed duplicates, added a fixed one as ui test and update one ui test to link to the corresponding ice ticket.
lets give this another go
@bors r=jieyouxu

**Comment by workingjubilee** [comment]
blaming this one on the internet

@bors retry

**Pull request merged**